service/project.go

```diff
@@ -1009,3 +1015,47 @@ func verifyAliasExists(alias, projectId string, newAliasDefinitions []model.Proj
 	}
 	return false, nil
 }
+
+func bbProjectIsValid(projName string, proj evergreen.BuildBaronSettings) error {
+	webHook, _, err := model.IsWebhookConfigured(projName, "")
+	if err != nil {
+		return err
+	}
+	webhookConfigured := webHook.Endpoint != ""
+
+	if !webhookConfigured && proj.TicketCreateProject == "" && len(proj.TicketSearchProjects) == 0 {
+		return nil
+	}
+	if !webhookConfigured && len(proj.TicketSearchProjects) == 0 {
+		return errors.Errorf("Must provide projects to search")
+	}
+	if proj.BFSuggestionServer != "" {
+		if _, err = url.Parse(proj.BFSuggestionServer); err != nil {
+			return errors.Errorf("Failed to parse bf_suggestion_server for project '%s'", projName)
```

Comment:
can we make all of the errors in this loop more human readable? what does BF suggestion server show up as for the UI?

Reply:
The old UI does not have any inputs right now for BF suggestion server (because no project had anything for it). These properties were migrated over as part of the plugin work but I'm actually not sure if they are used at all anymore - there is not any use of them in the evergreen repo aside from this validation that existed. I can create another ticket to investigate this and remove them if needed.